web/js/playground.js

```diff
@@ -33,22 +33,16 @@ function saveRecentlyUsedTools(recentTools) {
  * Update recently used tools, move the clicked tool to the top.
  */
 function updateRecentlyUsedTools(clickedTool) {
-    // const recentTools = getRecentlyUsedTools();
-    // const existingIndex = recentTools.findIndex(tool => tool.label.toLowerCase() === clickedTool.label.toLowerCase());
-
-    // if (existingIndex !== -1) {
-    //     recentTools.splice(existingIndex, 1);
-    // }
     console.log("updateRecentlyUsedTools called with:", clickedTool);
     let recentTools = getRecentlyUsedTools();
 
     // Remove the tool if it exists in recentTools, then add it to the top
     recentTools = recentTools.filter(tool => tool.label.toLowerCase() !== clickedTool.label.toLowerCase());
     recentTools.unshift(clickedTool);
 
-    const topThreeTools = recentTools.slice(0, 3);
-    saveRecentlyUsedTools(topThreeTools);
-    console.log("updated recent tools after click:", topThreeTools);
+    //const topThreeTools = recentTools.slice(0, 3);
+    saveRecentlyUsedTools(recentTools.slice(0, 3));
```

Comment:
I suppose this is a way to keep track of the top tools, if it works as expected.